### ISSUE TITLE
Update Cloud Non-Technical Explainer

### ISSUE THREAD

**Issue by emilyjones430** [bcgov/cloud-pathfinder-business-process#185]
**Describe the issue**
The cloud non-technical explainer was last updated in May. It will need to be updated to reflect the change of direction on the Early Adopter Program, focus and deliverable from Cloud Pathfinder Team as well as some of the COVID work. This document features in the Service Blueprint, and as such should be updated to reflect current priorities. 

**Which Sprint Priority is this issue related to?**
The 'Milestone' should clearly list Sprint Priorities - which one is this issue related to?

**Additional context**
N/A

**Definition of done**
- The explainer should be fully updated with the current work of the Pathfinder team, and fit in with the Service Blueprint that is being developed. 
- Reviewed by internal CP leadership 
- Determine which communication medium is best and integrate into that.

**Comment by emilyjones430** [comment]
From discussion with Gen and Stu, my task was to try to incorporate content from the non-technical explainer into the Early Adopters deck. I have sent a recommendation to @glam-bert that the content not be added due to the information being non-technical in nature - by the time teams are meeting with us about EA, they are already familiar with the benefits and challenges of cloud. Stu supported this approach. 

Gen had mentioned the possibility of having some of the content from the explainer used in one of the Playbooks that are being developed through HADF. Stu checked in with Greg to ask if his team was planning to develop a Cloud Playbook, and received the response that while it isn't on their list specifically, there may be opportunities down the road. 

Current recommendation to leave explainer as is, and revisit in the future as needed.

**Comment by sid-gov** [comment]
@ActionAnalytics had initial discussions with HADF developer and agreed to circle back on potential Cloud playbooks once a MVP is available to study as a use-case. The main issue right now is the WIP stability of the Demo app and lack of existing Early Adopter (EA) examples.

**Comment by emilyjones430** [comment]
I've finished the first draft of the incorporated Secure Cloud web content, FAQs and non-technical explainer. This combines the current Secure Cloud webpage content with sections of the FAQ (including guidance and contacts for privacy, security and procurement) and the non-technical explainer (mainly content around cloud economics). I've also included references to Corporate Software Asset Management and Communications and Collaboration Services (OCIOO365@gov.bc.ca - Dennis Murch's group for assistance with Microsoft products).

Based on the interregnum, no website updates will be allowed in next few months. As such, we will wait to move this through approvals closer to when the content could be posted. I have added a card under "new issues" (#228) in regards to the review and approval of the content.
